client-device-connection-infinispan/pom.xml

```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I'm not sure this module should be the "root" one of the whole Hono project. I'm not sure yet what would be the best place for it. Maybe put it in the "client" one, or as a part of the service implementation. Or even refactor the client one to have submodules like adapters and services.

Reply:
We definitely need to refactor the client module. For the time being, I did not want to put it in there because it has a dependency on `service-base`, which `client` doesn't have so far. But I agree, this is far from optimal and we might be able to come up with a better layout if we move some of the more generic interfaces from service-base to core. Maybe we can talk about this next week at BCX.

Reply:
Sure. The only concern is that once we release with the module having it's maven coordinates defined. Later changes would be braking.

Reply:
This module is only used by the protocol adapters anyway so I do not see a big issue there.